### PR TITLE
share.deps: kill serialport server before install finished

### PR DESCRIPTION
**Pull request by xutian**
kill serailport.exe to release serialport to finished program
to write string to serialport.

Signed-off-by: Xu Tian <xutian@redhat.com>

### CODE REVIEW

File: shared/deps/finish/finish.bat [hunk 1]
```diff
@@ -1,3 +1,5 @@
+taskkill /IM serialport.exe /F
```
Comment (yanan-fu) [inline shared/deps/finish/finish.bat]: At last,  don't need to start the serial port again ?

Comment (xutian) [inline shared/deps/finish/finish.bat]: No, it will start after guest reboot in last line

**Comment by xutian** [comment]
@yanan-fu  what's your test results ?

**Comment by yanan-fu** [comment]
Test PASS.
Acked-by: Yanan Fu yfu@redhat.com

**Comment by xutian** [comment]
thanks @yanan-fu

**Pull request merged**